IonDotnet/Internals/Text/TextScanner.cs

```diff
@@ -1884,17 +1907,27 @@ private int ReadTripleQuotedChar(bool isClob)
                 case '\'':
                     if (Is2SingleQuotes())
                     {
+                        // clobs disallow comments everywhere within the value
+                        var commentStrategy = isClob ? CommentStrategy.Error : CommentStrategy.Ignore;
+
                         // so at this point we are at the end of the closing
                         // triple quote - so we need to look ahead to see if
                         // there's just whitespace and a new opening triple quote
-                        c = SkipOverWhiteSpace(CommentStrategy.Ignore);
+                        c = SkipOverWhiteSpace(commentStrategy);
                         if (c == '\'' && Is2SingleQuotes())
                         {
                             // there's another segment so read the next segment as well
                             // since we're now just before char 1 of the next segment
                             // loop again, but don't append this char
                             return CharacterSequence.CharSeqStringNonTerminator;
                         }
+                        // at this point, we are at the end of the closing
+                        // triple quote and it does not follow by any other '
+                        // so the only acceptable charcter is the closing brace
+                        if (isClob && c != '}')
+                        {
+                            throw new IonException($"Bad Character: { ((char)c) } , expected \"}}}}\"");
```

Comment:
There should just be two `}` after "expected", right? Also, I'd recommend saying `Bad character in Clob:...` since that's useful information to the user.

Reply:
Oh, are the extra `}` characters used for escaping? If so, disregard that aspect of my comment.

Reply:
done.